sys/include/net/telnet.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2021 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_telnet basic Telnet server implementation
+ * @ingroup     net_ipv6
+ * @brief       Telnet server functions
+ * @{
+ *
+ * @file
+ * @brief   Telnet server ([RFC 855](https://tools.ietf.org/html/rfc855)) implementation
+ * @note    This implementation only supports a single client
```

Comment:
You could use `sock_async_tcp` to support multiple, no?

Reply:
Maybe, but I want this as a debugging tool, so let's keep things simple.
If someone wants to run a MUD, we can add that later :wink: